src/System.Device.Ports/System.Device.Ports.SerialPort/SerialData.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Device.Ports.SerialPort
+{
+    /// <summary>
+    /// Specifies the type of character that was received on the serial port of the SerialPort object.
+    /// </summary>
+    public enum SerialData
+    {
+        /// <summary>
+        /// A character was received and placed in the input buffer.
+        /// In the Windows implementation, this is EV_RXCHAR.
```

Comment:
I'd leave platform implementation details out of here

Reply:
Agree. I just ported the class in the project to resolve other issues in the first place.